ui/src/routes/devices.$id.settings.general._index.tsx

```diff
@@ -92,6 +92,21 @@ export default function SettingsGeneralRoute() {
               />
             </SettingsItem>
           </div>
+
+          <div className="mt-2 flex items-center justify-between gap-x-2">
+            <SettingsItem
+              title="Reboot Device"
+              description="Power cycle the JetKVM"
```

Comment:
It's not actually power-cycling, though - it's only rebooting the OS - not the same.